src/libraries/System.Configuration.ConfigurationManager/src/System/Configuration/Internal/DelegatingConfigHost.cs

```diff
@@ -231,12 +231,12 @@ public virtual void RefreshConfigPaths()
 
         public virtual IDisposable Impersonate() => new DummyDisposable();
 
-#pragma warning disable SYSLIB0003 // Obsolete: CAS
+#if NETFRAMEWORK // Obsolete: CAS (.NET Framework only)
```

Comment:
This source file should not be included for .NETFramework at all.  .NETFramework should just be type-forwards.  Same goes for all the `src` files.  Just delete the implementation.  Keep the test ifdef though.

Reply:
ah ok